platform/smart-home-app.ts

```diff
@@ -3,30 +3,46 @@
  */
 /// <reference types="@google/local-home-sdk" />
 
-import { App } from '@google/local-home-sdk';
+import { DeviceManagerStub } from './device-manager';
+import { MockLocalHomePlatform } from './mock-local-home-platform';
+import { MockNetwork } from './mock-radio';
 
-//TODO(cjdaly) fill in implementation of these methods
-class AppStub implements App {
+export class AppStub implements smarthome.App {
   private version: string;
-  private deviceManager: any;
+  public identifyHandler: smarthome.IntentFlow.IdentifyHandler;
+  public executeHandler: smarthome.IntentFlow.ExecuteHandler;
+  public reachableDevicesHandler: smarthome.IntentFlow.ReachableDevicesHandler;
+
   constructor(version: string) {
     this.version = version;
+    //  Allows Local Home Platform to access handlers
+    MockLocalHomePlatform.getInstance().setApp(this);
   }
+
   getDeviceManager(): smarthome.DeviceManager {
-    return;
+    return MockLocalHomePlatform.getInstance().getDeviceManager();
   }
+
   listen(): Promise<void> {
-    return;
+    return Promise.resolve();
```

Comment:
Is this method to be implemented later?

Reply:
Good call.  I suppose the intended behavior is that `listen()` notifies the Local Home Platform that the Local Fulfillment app has set all the handlers and is ready to receive requests.  I added a flag so that developers can assert `listen()` was called.  Also added an example of this flag being asserted.